MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/module/loader/LibraryLoaderProviderImpl.java

```diff
@@ -41,9 +40,8 @@ public void load(String name) {
         init(Mapbox.getApplicationContext(), false);
         loadLibrary(name);
       } catch (MapboxConfigurationException exception) {
-        Logger.e(TAG, "Couldn't load so file with relinker, application context missing, "
-          + "call Mapbox.getInstance(Context context, String accessToken) first");
-        throw new UnsatisfiedLinkError(exception.getMessage());
+        //Rollback to System.loadLibrary if SoLoader failed to load library.
+        System.loadLibrary(name);
```

Comment:
This code block won't be hit when SoLoader fails to load (we would hit a UnsatisfiedLinkError not a MapboxConfgurationException)